kubernetes-elasticsearch-cluster/es-data-stateful.yaml

```diff
@@ -48,24 +50,32 @@ spec:
         - name: NODE_INGEST
           value: "false"
         - name: HTTP_ENABLE
-          value: "false"
+          value: "true"
         - name: ES_JAVA_OPTS
-          value: -Xms1g -Xmx1g
+          value: -Xms2g -Xmx2g
         - name: PROCESSORS
```

Comment:
[Port-forwarding no longer works.](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#index-in-gke) After the `kubectl port-foward` command, if I open up a Chrome tab with `http://localhost:9200/_cat/indices?v&s=index`, I get connection refused.

Perhaps putting [NETWORK_HOST which used to be in es-client.yaml](https://github.com/DataBiosphere/data-explorer-indexers/blob/master/kubernetes-elasticsearch-cluster/es-client.yaml#L51) would fix this, as well as not being able to curl localhost:9200.

Reply:
Done (will test out localhost:9200, I bet you're right that's the same thing).

Reply:
Just validated it fixes port forwarding too.